packages/hyperflux/functions/ActionFunctions.ts

```diff
@@ -220,6 +220,8 @@ export type PartialActionType<Shape extends ActionShape<any>> = Omit<
  * @param actionShape
  * @returns a function that creates an instance of the defined action
  */
+export const actionDefinitions = {} as Record<string, any>
```

Comment:
`a` should be capitalized as `ActionDefinitions`

Reply:
roger that

Reply:
done